component/uiextension/src/main/java/org/exoplatform/management/uiextension/Utils.java

```diff
@@ -232,4 +259,12 @@ private static boolean equals(String[] sourcePermissions, String[] targetPermiss
       }
     }
   }
+  private static  boolean notAdded(String path, List<NodeComparison> comparisons){

```

Comment:
No need to add this method, just implement the equal function in NodeComparison.java and use List<comparison>.contains(comparison)